<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>521</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>56</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Sitio Web</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="led_url">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>471</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string extracomment="http://"/>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_w">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Realizar pausa de 20 segundos</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_r">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Descargar de manera Recursiva</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_k">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Convertir Enlaces</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_p">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Descargar imagenes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_u">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Emular Navegador Web</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_l">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Limitar ratio de descarga</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="led_destino">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>321</width>
       <height>23</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_seleccionar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Carpeta de Destino</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_descargar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>94</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Descargar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_detener">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>270</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Detener Descarga</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
